### DISCUSSION TITLE
How to get rid of 500 error on Hiddify? | چگونه مشکل خطای ۵۰۰ را رفع کنیم

### DISCUSSION THREAD

**Discussion by lymanjre** [hiddify/Hiddify-Manager#782] [Ideas ایدهها]
How to get rid of 500 error on Hiddify? | چگونه مشکل خطای ۵۰۰ را رفع کنیم

**Comment by botgame96** [comment]
<div dir=rtl>

**آموزش رفع ارور 500 بعد از آپدیت جدید در پنل هیدیفای** 

- گام اول : 

ابتدا با ترمینال وارد سرور مجازی خود میشیم =>پنجره ای که پنل هیدیفای را در محیط vps نشان می دهد  .

</div>

<div align=center>

![0uYKz](https://user-images.githubusercontent.com/125398461/233766198-0a8818a5-2f49-4c81-b716-e05541a3b15b.jpg)

</div>

<div dir=rtl>

- گام دوم :

 بعد از متصل شدن به سرور مجازی از طریق ترمینال؛ حالا باید کش مرورگر که از کوکی‌های جدید پر شده را خالی کنیم .

اگر نمی‌دانید چطور کش مرورگر را پاک کنید کافیست در گوگل سرچ کنید:

</div>

<div dir=ltr>
How to clear cash of <نام مرورگر>



- گام سوم : 

بر می‌گردیم به vps حالا روی  update می‌زنیم و در صفحه بعد بر روی default می‌زنیم .

</div>

<div align=center>

![0up8b](https://user-images.githubusercontent.com/125398461/233766282-4189761d-d557-4760-b691-0afdeab0f06a.jpg)
<br>
![0uDr7](https://user-images.githubusercontent.com/125398461/233766312-05465f5c-46cb-4ba5-9558-20e60cafff9f.jpg)
</div>

<div dir=rtl>

- گام چهار : 

در cdn  کلودفلر بخش  speed > optimization >autominify تیک ها را چک کنید که انتخاب شده نباشند مانند تصویر زیر .

</div>

<div align=center>

![0udGJ](https://user-images.githubusercontent.com/125398461/233766341-3ea0d646-0101-4652-82da-1201ff90eaa6.jpg)

</div>

<div dir="rtl">

- گام پنج :

 حالا یکبار دیگر از putty بر روی سرور خود نصب مجدد را می‌زنیم    .

</div>

<div align=center>

![0u6Vr](https://user-images.githubusercontent.com/125398461/233766402-21a3492d-e54c-494c-90b8-99c8afe5c2ef.jpg)


</div>
 
نکته : این روش برای زمانی که خطای ۵۰۰ مکرر تکرار می‌شود ( شاید پس از آنکه آپدیت کردید) کار ساز بوده و و شما می‌توانید از لینک‌ها استفاده کامل را ببرید و وارد پنل شوید .
